### PR TITLE
OPENTOK-41606: Update README with code changes

### PR DESCRIPTION
**Pull request by v-kpheng**
The README changes code snippets that were removed from the reference code during refactoring.  This PR updates the README with the code currently in use.  Also, some light editing was done as well.

### CODE REVIEW

File: Simple-Multiparty/README.md [hunk 1]
```diff
@@ -4,50 +4,24 @@ Note: Read the README.md file in the Basic-Video-Chat folder before starting her
...
-        toggleAudio.setVisibility(View.VISIBLE);
-    }
+@Override
+public void onStreamReceived(Session session, Stream stream) {
+    Log.d(TAG, "onStreamReceived: New stream " + stream.getStreamId() + " in session " + session.getSessionId());
+
+    final Subscriber subscriber = new Subscriber.Builder(MainActivity.this, stream).build();
+    mSession.subscribe(subscriber);
+    addSubscriber(subscriber);
+}
```
Comment (msach22) [inline Simple-Multiparty/README.md:18]: Should we detail what `addSubscriber` does since the view is actually updated there now?

Comment (v-kpheng) [reply Simple-Multiparty/README.md:18]: I'm leaning towards no.  Reason being, the previous code, which was lengthier, didn't add any details describing that each portion of the code did.  (The preceding passage seemed sufficient).

That said, what did you have in mind?

Comment (msach22) [reply Simple-Multiparty/README.md:18]: Sounds good - just wanted to get an opinion.

**Review by msach22 — APPROVED** [review]
LGTM

**Pull request merged**